packages/db/src/migrations/20240728101230_add_table_indexes.ts

```diff
@@ -24,15 +24,6 @@ export const up = async (knex: Knex): Promise<void> => {
   await knex.raw('CREATE INDEX CONCURRENTLY IF NOT EXISTS providers_updated_at_index ON providers(updated_at)');
 
   await knex.raw('CREATE INDEX CONCURRENTLY IF NOT EXISTS rules_updated_at_index ON rules(updated_at)');
-
```

Comment:
We didn't flag these initially but I think should also be removed.

Reply:
I ran a search for `VACUUM` in all .ts files and didn't find any additional.

Reply:
Yeah, the sql files in the migrations directory aren't meant to be run as part of a deployment. Those are for operators to execute manually.